ci/builders/web_engine.json

```diff
@@ -0,0 +1,100 @@
+[
+    {
+        "archives": [],
+        "drone_dimensions": [
+            "device_type=none",
+            "os=Windows-10"
+        ],
+        "gn": [
+            "--runtime-mode",
+            "debug",
+            "--unoptimized",
+            "--full-dart-sdk",
+            "--prebuilt-dart-sdk"
+        ],
+        "name": "windows_host_debug",
+        "ninja": {
+            "config": "host_debug_unopt",
+            "targets": []
+        },
+        "platform": "Windows",
+        "tests": []
+    },
+    {
+        "archives": [],
+        "drone_dimensions": [
+            "device_type=none",
+            "os=Linux"
+        ],
+        "generators": {
```

Comment:
nit: can generators be moved after ninja so that the config can be read in order things execute?

Reply:
done!